pylettize/cli.py

```diff
@@ -0,0 +1,83 @@
+"""
+Pylettize. Image re-palettizer written in Python.
+
+Palettes:
+    Any text file with line-separated hex-colors can be supplied as the palette.
+    In addition, the following named default palettes are available
+    * gruvbox    The classic Gruvbox color scheme
+    * obama      Colors from the "Hope" posters
+    * primaries  RGB and CMY
+
+Blending strategies:
+    Pylettize provides two main blending strategies
+    * hard  Using a binary mapping from input pixel color to palette color
+    * soft  Using a linear combination of palette colors in each pixel
+            The degree of "softness" is determined with the -T temperature option
+
+Usage:
+    pylettize -h
+    pylettize hard <img> [-p] <palette> [-o <output>]
+    pylettize soft <img> [-p] <palette> [-o <output> -T <temperature>]
+
+-h, --help                                     Show this screen
+-p, --palette-file                             Use a provided text file as palette
```

Comment:
Maybe the short variant here should be `-f` for file? I'm not sure though

Reply:
I would say either. -p is not commonly used so it cannot be misinterpreted.